src/ssl.c

```diff
@@ -48623,8 +48808,8 @@ void wolfSSL_OPENSSL_config(char *config_name)
 #endif /* !NO_WOLFSSL_STUB */
 #endif /* OPENSSL_ALL || WOLFSSL_NGINX || WOLFSSL_HAPROXY */
 
-#if defined(OPENSSL_ALL) || defined(WOLFSSL_NGINX) || defined(WOLFSSL_HAPROXY) \
-    || defined(OPENSSL_EXTRA) || defined(HAVE_LIGHTY)
+#if defined(HAVE_EX_DATA) || defined(OPENSSL_ALL) || defined(WOLFSSL_NGINX) || \
```

Comment:
This change also needs to be done in ssl.h for `wolfSSL_X509_get_ex_new_index`

Reply:
Turned out the gate for `wolfSSL_X509_get_ex_new_index()` was fine as-was (reverted in ssl.c, left as-was in ssl.h).

The `wolfSSL_get_ex_data` symbol failure you report above was due to a bad gate on its definition, which is now fixed.